### DISCUSSION TITLE
Legend backgroud color

### DISCUSSION THREAD

**Discussion by AlexStox** [highfestiva/finplot#191] [Q&A]
Hi Jonas, I examined your [SP500 example](https://github.com/highfestiva/finplot#custom-crosshair-and-legend) and even the _init_ but didn't understand **how to change the background color for legend**. 

Can you help me please? 
Should I add additional "style" parameter into the **fmt** string?
    `fmt = '<span style="color:#%s">%%.2f</span>' % ('0b0' if (row.Open<row.Close).all() else 'a00')`

**Comment by highfestiva** [comment]
`fplt.legend_fill_color = '#ff00ff`
